versions.cfg

```diff
@@ -61,7 +61,7 @@ plone.recipe.alltests = 1.5
 plone.recipe.zeoserver = 1.3
 plone.recipe.zope2instance = 4.2.22
 plone.releaser = 1.5.4
-plone.versioncheck = 1.5.1
+plone.versioncheck = 
```

Comment:
Indeed, please keep the version pinned. If plone.versioncheck is added to auto-checkout it overrides the version pin anyway.

Also I prefer to not have WIP in title but a proper use of the labels ``status: in-progress``